mongo_connector/oplog_manager.py

```diff
@@ -479,26 +531,84 @@ def dump_collection(self):
             return None
         long_ts = util.bson_ts_to_long(timestamp)
 
-        dump_set = self.namespace_set or []
-        LOG.debug("OplogThread: Dumping set of collections %s " % dump_set)
-
-        # No namespaces specified
-        if not self.namespace_set:
-            db_list = retry_until_ok(self.primary_client.database_names)
-            for database in db_list:
-                if database == "config" or database == "local":
-                    continue
-                coll_list = retry_until_ok(
-                    self.primary_client[database].collection_names)
-                for coll in coll_list:
-                    # ignore system collections
-                    if coll.startswith("system."):
-                        continue
-                    # ignore gridfs collections
-                    if coll.endswith(".files") or coll.endswith(".chunks"):
+
+        def get_dump_set():
```

Comment:
Since this is only ever called from one place and doesn't take any arguments, it doesn't need to be a function. We can just modify `dump_set` directly.